chroma-bundles/include/find_deps.py

```diff
@@ -91,7 +91,7 @@ def disable_all_repos(self):
                repo.id != "extras" and repo.id != "extras-centos7.3-x86_64" and \
                repo.id != "addon-epel7-x86_64" and repo.id != "epel" and \
                repo.id != "ngompa-dnf-el7" and \
-               repo.id != "managerforlustre-manager-for-lustre" and \
+               repo.id != "wcjohnso-manager-for-lustre" and \
```

Comment:
You should not really need to add your private repo here.  But if you were going to anyway, you should add yours to this list, not replace the standard one.